lava-slave/scripts/start.sh

```diff
@@ -16,6 +16,8 @@ sleep 2
 for item in $(ls /etc/conmux/*cf)
 do
 	echo "Add $item"
+	# On some OS, the rights/user from host are not duplicated on guest
+	grep -o '/dev/[a-zA-Z0-9_-]*' $item | xargs chown uucp
```

Comment:
This should only touch /dev/tty*, it currently changes /dev/*

Reply:
it greps the serialname in config so it wont touch /dev/*. Furthermore, the serialname never begin by tty since it is the board name.